models/classes/LtiProvider/LtiProviderFieldsMapper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoLti\models\classes\LtiProvider;
+
+use oat\oatbox\service\ConfigurableService;
+
+class LtiProviderFieldsMapper extends ConfigurableService
+{
+    public const SERVICE_ID = 'taoLti/LtiProviderFieldsMapper';
+    public const OPTION_MAP = 'map';
+
+    public function map(string $rdfUri): ?string
+    {
+        return $this->getMap() [$rdfUri] ?? null;
+    }
+
+    private function getMap(): array
+    {
+        return $this->getOption(self::OPTION_MAP) ?? [];
+    }
```

Comment:
```suggestion
    public function map(string $rdfUri): ?string
    {
        return $this->getOption(self::OPTION_MAP)[$rdfUri] ?? null;
    }
```

Since you are using `??` it will ignore any error for any level of index, so you do not need to use it twice.

Reply:
But take a look at this comment too: https://github.com/oat-sa/extension-tao-lti/pull/274/files#r491025056

It should be justifiable to have a service configurable (by `.conf.php` file). And I think it is not the case here.